websites/R/Rust Lang Book/presence.ts

```diff
@@ -0,0 +1,41 @@
+const presence = new Presence({
+	clientId: "1033608073106968576",
+});
+
+presence.on("UpdateData", async () => {
+	const path = window.location.pathname
+			.replace("/book/", "")
+			.replace(".html", ""),
+		presenceData: PresenceData = {
+			largeImageKey: "rust-logo-512",
```

Comment:
Imgur links are preferred over Discord asset names. This applies to `smallImageKey` as well.
```suggestion
			largeImageKey: "https://i.imgur.com/kTnc40Q.png",
```